### ISSUE TITLE
Exported value: hostname not exported for the resource azurerm_redis_cache

### ISSUE THREAD

**Issue by horsey** [hashicorp/terraform#14127]
### Terraform Version
v0.9.2

### Affected Resource(s)
- azurerm_redis_cache

### Terraform Configuration Files
```hcl
resource "azurerm_redis_cache" "redis0" {
    name                = "redis0"
    location            = "East US"
    resource_group_name = "xxxx"
    capacity            = 1
    family              = "C"
    sku_name            = "Standard"
    enable_non_ssl_port = false

    redis_configuration {
        maxclients = "1000"
    }

    tags {
        role = "redis"
        environment = "production"
    }
}
 output "redis-host" {
        value = "${azurerm_redis_cache.redis0.hostname}"
  }


```

### Debug Output
None

### Panic Output
None
### Expected Behavior
The output value of `redis-host` should be displayed at the end of `terraform apply` 
Ref: [https://www.terraform.io/docs/providers/azurerm/r/redis_cache.html](url)

### Actual Behavior
The output value of `redis-host` is not displayed at the end of `terraform apply`

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
None
### References
None.

**Comment by tombuildsstuff** [comment]
Hi @horsey 

Thanks for raising this issue.

From what I can see, this issue was fixed in PR #13650, which was released in Terraform 0.9.4. Would it be possible to upgrade and confirm if this solves the issue?

Using the config:
```
resource "azurerm_resource_group" "test" {
  name     = "tharvey-bug14127"
  location = "West Europe"
}

resource "azurerm_redis_cache" "redis0" {
  name                = "tharvey-bug14127redis"
  location            = "${azurerm_resource_group.test.location}"
  resource_group_name = "${azurerm_resource_group.test.name}"
  capacity            = 1
  family              = "C"
  sku_name            = "Standard"
  enable_non_ssl_port = false

  redis_configuration {
    maxclients = "1000"
  }

  tags {
    role        = "redis"
    environment = "production"
  }
}

output "redis-host" {
  value = "${azurerm_redis_cache.redis0.hostname}"
}
```

I get the output:
```
$ envchain azurerm terraform apply
[WARN] /Users/tharvey/code/bin/terraform-provider-azurerm overrides an internal plugin for azurerm-provider.
  If you did not expect to see this message you will need to remove the old plugin.
  See https://www.terraform.io/docs/internals/internal-plugins.html
azurerm_resource_group.test: Refreshing state... (ID: /subscriptions/c0a607b2-6372-4ef3-abdb-...7b56ba/resourceGroups/tharvey-bug14127)
azurerm_redis_cache.redis0: Creating...
  capacity:                                 "" => "1"
  enable_non_ssl_port:                      "" => "false"
  family:                                   "" => "C"
  hostname:                                 "" => "<computed>"
  location:                                 "" => "westeurope"
  name:                                     "" => "tharvey-bug14127redis"
  port:                                     "" => "<computed>"
  primary_access_key:                       "" => "<computed>"
  redis_configuration.#:                    "" => "1"
  redis_configuration.0.maxclients:         "" => "1000"
  redis_configuration.0.maxmemory_delta:    "" => "<computed>"
  redis_configuration.0.maxmemory_policy:   "" => "volatile-lru"
  redis_configuration.0.maxmemory_reserved: "" => "<computed>"
  resource_group_name:                      "" => "tharvey-bug14127"
  secondary_access_key:                     "" => "<computed>"
  sku_name:                                 "" => "Standard"
  ssl_port:                                 "" => "<computed>"
  tags.%:                                   "" => "2"
  tags.environment:                         "" => "production"
  tags.role:                                "" => "redis"
azurerm_redis_cache.redis0: Still creating... (10s elapsed)
azurerm_redis_cache.redis0: Still creating... (20s elapsed)
azurerm_redis_cache.redis0: Still creating... (30s elapsed)
azurerm_redis_cache.redis0: Still creating... (40s elapsed)
azurerm_redis_cache.redis0: Still creating... (50s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (1m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (2m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (3m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (4m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (5m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (6m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (7m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (8m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (9m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (10m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (11m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m0s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m10s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m20s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m30s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m40s elapsed)
azurerm_redis_cache.redis0: Still creating... (12m50s elapsed)
azurerm_redis_cache.redis0: Still creating... (13m0s elapsed)
azurerm_redis_cache.redis0: Creation complete (ID: /subscriptions/c0a607b2-6372-4ef3-abdb-...soft.Cache/Redis/tharvey-bug14127redis)

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

The state of your infrastructure has been saved to the path
below. This state is required to modify and destroy your
infrastructure, so keep it safe. To inspect the complete state
use the `terraform show` command.

State path:

Outputs:

redis-host = tharvey-bug14127redis.redis.cache.windows.net
```

Thanks!

**Comment by horsey** [comment]
@tombuildsstuff 
Thanks for the response.  I think that the PR #13650 fixes this issue. I was planning to submit a patch with the fix, but it is already fixed! 
Happy to upgrade to v0.9.4.
